packages/fluent-editor/src/modules/file/modules/file-module.ts

```
@@ -54,4 +54,4 @@ export class FileModule {
       alink?.parentNode?.removeChild(a)
     }
   }
-}
+}
```

ESLint

[error] Newline required at end of file but not found.

(style/eol-last)